quickstart-templates/workload-identity-creates-workload-identities/README.md

```diff
@@ -0,0 +1,70 @@
+# Create Workload Identity Federation that can create other workload identity federations
+
+## The Idea
+
+Restrict and limit the need to use high human privileges to create resources.
```

Comment:
I know what high privileges are, but what are "high human privileges"?  Is that a way to indicate high privileges in delegated scenarios?

Reply:
I rephrased a bit.